### DISCUSSION TITLE
How to Call And Play Audio after the call is established?

### DISCUSSION THREAD

**Discussion by joaquinjsb** [baresip/baresip#2874] [Q&A] [answered]
Hello, 
I'm trying to make an automatic call system, but I'm struggling to play the audio only after the call has been established, I launch baresip following these steps:

1. start baresip
2. call using the /dial command
3. after the call is established manually, I just trigger the /ausrc aufile,/usr/local/share/baresip/myaudio.wav

so my question is,

1.  how can I loop the audio or start it only after the call has been answered?  when I use the /ausrc after dialing, it just likes play it instantly
2. how do I automate it? I wanted to do it via the https interface, but there's no documentation , otherwise, imma gonna do it through shell emulation but I would prefer not doing that.

**Comment by cspiel1** [comment]
Please could you try to set the audio source before initiating the call with `/dial`?

**Reply by joaquinjsb** [reply]
That's what I did, but the audio starts even before the call has been answered so maybe if the phone rings for a while maybe the audio is over, I would need it to start when the call is answered or at least make it loop

**Reply by cspiel1** [reply, marked as answer]
I'll try to reproduce this soon.

**Comment by cspiel1** [comment]
The loop is not supported. Currently, the only way is to periodically invoke `/ausrc aufile,...`.

I created an issue out of this discussion: https://github.com/baresip/baresip/issues/2882
